os/include/tinyara/pm/pm.h

```diff
@@ -252,6 +252,7 @@
 /* This enumeration provides all power management domains */
 enum pm_domain_e {
 	PM_IDLE_DOMAIN = 0,
+	PM_LCD_DOMAIN = 1,
```

Comment:
```
#ifdef CONFIG_LCD
...
#endif
```

Reply:
The PM merely provides the domain to other functions.

the memory and operating time are the same, there is no need to provide the domain by checking whether the config is enabled or disabled

This is the dependence on the user area.